docs/image.md

```diff
@@ -424,6 +424,8 @@ This prop can also contain several remote URLs, specified together with their wi
 
 The currently supported formats are `png`, `jpg`, `jpeg`, `bmp`, `gif`, `webp`, `psd` (iOS only). In addition, iOS supports several RAW image formats. Refer to Apple's documentation for the current list of supported camera models (for iOS 12, see https://support.apple.com/en-ca/HT208967).
 
+> Note: WebP support on iOS is only available starting from iOS 14. If your app targets a lower iOS version other than iOS 14 please consider adding a fallback image with format other than WebP.
```

Comment:
Or you could use a custom RCTImageDataDecoder instead of a fallback image. That way you can use webp images for older iOS versions.

Reply:
@danilobuerger +1 on `RCTImageDataDecoder` but as with many thing with React Native tutorial on how to use a custom  `RCTImageDataDecoder` is not yet available in the docs so i am a bit on the edge wether to just mention this on the notes but without any references on how to exactly do this.